microprofile/metrics-smallrye/src/main/resources/subsystem-templates/microprofile-metrics-smallrye.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version='1.0' encoding='UTF-8'?>
+
+<!--  See src/resources/configuration/ReadMe.txt for how the configuration assembly works -->
+<config>
+    <extension-module>org.wildfly.extension.microprofile.metrics-smallrye</extension-module>
+    <subsystem xmlns="urn:wildfly:microprofile-metrics-smallrye:1.0"
+               security-enabled="false"
```

Comment:
Should this be `true`? The default for in the XSD and on the attribute is `true`.

Reply:
yes, the setting is correct.
By default, security is enabled. However for our standalone configuration where the management HTTP interface is split from the application HTTP interface we disable authentication.